# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296936488 28800
#      Sat Feb 05 12:08:08 2011 -0800
# Branch release-3-4-x
# Node ID 3e34ad1d31df21e929d1f92d883fd816e2818878
# Parent  929cd1a6754be5be319e9c415e03c050edad5832
Update copyright on written documentation to 2011

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-05  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/octave.texi: Update copyright on manual to 2011.
+
 2010-02-05  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/mkoctfile.1, interpreter/octave-config.1,
 	interpreter/octave.1: Overhaul man pages for release 3.4.
 
 2010-02-04  Rik  <octave@nomad.inbox5.com>
 
 	* icons/octave-logo.ico: Add logo format left out of last changeset.
diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -79,17 +79,17 @@ END-INFO-DIR-ENTRY
 @c texinfo-multiple-files-update.
 @include version.texi
 
 @settitle GNU Octave
 
 @ifnottex
 
 Copyright @copyright{} 1996, 1997, 1999, 2000, 2001, 2002, 2005, 2006,
-2007 John W. Eaton.
+2007, 2011 John W. Eaton.
 
 Permission is granted to make and distribute verbatim copies of
 this manual provided the copyright notice and this permission notice
 are preserved on all copies.
 
 @ignore
 Permission is granted to process this file through Tex and print the
 results, provided the printed document carries copying permission
@@ -123,17 +123,17 @@ modified versions.
 @titlefont{Free Your Numbers} 
 @end multitable 
 @author John W. Eaton
 @author David Bateman
 @author S@o{}ren Hauberg
 @page
 @vskip 0pt plus 1filll
 Copyright @copyright{} 1996, 1997, 1999, 2000, 2001, 2002, 2005, 2006,
-2007 John W. Eaton.
+2007, 2011 John W. Eaton.
 
 This is the third edition of the Octave documentation, and is consistent
 with version @value{VERSION} of Octave.
 
 Permission is granted to make and distribute verbatim copies of
 this manual provided the copyright notice and this permission notice
 are preserved on all copies.
 
